samples/cpp/embedded-speech/README.md

```diff
@@ -162,6 +162,20 @@ export CLOUD_SPEECH_SERVICE_REGION="westus"
 ```
 **Note:** On Linux and macOS, use quotation marks around the values of environment variables set on the command line as shown in the example. Otherwise strings with spaces (like embedded speech model/voice names) will be cut short.
 
+## Remarks
+
+For a new project from scratch, install the following Speech SDK packages as needed:
+* [Microsoft.CognitiveServices.Speech](https://www.nuget.org/packages/Microsoft.CognitiveServices.Speech)
+  * mandatory for using the Speech SDK
+* [Microsoft.CognitiveServices.Speech.Extension.Embedded.SR](https://www.nuget.org/packages/Microsoft.CognitiveServices.Speech.Extension.Embedded.SR)
+  * required for embedded speech recognition
+* [Microsoft.CognitiveServices.Speech.Extension.Embedded.TTS](https://www.nuget.org/packages/Microsoft.CognitiveServices.Speech.Extension.Embedded.TTS)
+  * required for embedded speech synthesis
+* [Microsoft.CognitiveServices.Speech.Extension.ONNX.Runtime](https://www.nuget.org/packages/Microsoft.CognitiveServices.Speech.Extension.ONNX.Runtime)
+  * required by embedded speech
```

Comment:
>required by embedded speech [](http://example.com/codeflow?start=4&length=27)

could be clearer when this is needed.  for any embedded speech, or only SR?

Reply:
Any "embedded speech" - I guess it could have "any" or both "SR and TTS".
On the other hand this is an automatic dependency for SR and TTS.

Reply:
Added "recognition and synthesis"